mods/playerapi.json

```diff
@@ -4,7 +4,35 @@
     "authors": ["Divisor"],
     "versions": [
         {
-            "name": "1.7",
+            "name": "1.2.5",
+            "mcvsn": ["1.2.5"],
+            "files": [
+                {
+                    "filename": "MC 1.2.5 - Player API client 1.5.zip",
+                    "ipfs": "QmUWgZN4yBKg31SKtiSCiXEfBu7hZrGDLF6gYbRjCf9cuD",
+                    "archived": "465d66e9c2ba0f2b661f638956986573d36c59b2397e9681cd5310124ed9f06a/MC 1.2.5 - Player API client 1.5.zip",
+                    "hash": { "type": "sha256", "digest": "465d66e9c2ba0f2b661f638956986573d36c59b2397e9681cd5310124ed9f06a" },
+                    "urls": [
+                        { "type": "page", "url": "http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1277996-player-api" },
+                        { "type": "original", "url": "http://www.mediafiregaming.com/file/3t4ohn1scod44rr/MC+1.2.5+-+Player+API+client+1.5.zip" }
+                    ]
+                },
```

Comment:
This JSON is malformed (doesn't parse correctly). Please run your JSON through a linter and make sure it parses correctly. Here is an online one: https://jsonlint.com/